src/parcsr_ls/par_mgr_rap.c

```diff
@@ -587,6 +587,14 @@ hypre_MGRBuildCoarseOperator(void                *mgr_vdata,
          hypre_ParCSRMatrixAdd(1.0, A_CC, -1.0, RAP_c, &RAP);
          hypre_ParCSRMatrixDestroy(RAP_c);
       }
+      else if (!Wr && Wp)
+      {
+         /* Prolongation is not the injection operator (Wp != NULL) and
+            Restriction is the injection operator (Wr == NULL) */
+         RAP_c = hypre_ParCSRMatMat(A_CF, Wp);
```

Comment:
@victorapm Wp here is not negated already right?

Reply:
Good point, right!

Reply:
For reference: we found different definitions of Wp and Wr throughout MGR's code (without the negative sign and with the negative sign). We decided to use a single convention that considers the negative sign already in Wp and Wr, so that prolongation is computed as `[Wp I]^T` and restriction as `[Wr I]`. The unification work was done in [89c2d0e](https://github.com/hypre-space/hypre/pull/1247/commits/89c2d0ef0e2112a644aef2842c1649f08d9dbc8e)